src/rustdoc-internals/rustdoc-test-suite.md

```diff
@@ -4,21 +4,19 @@ This page is specifically about the test suite named `rustdoc`, for other test s
 
 The `rustdoc` test suite is specifically used to test the HTML output of rustdoc.
 
-This is achived by means of `htmldocck.py`, a custom checker script that leverages [XPath].
+This is achieved by means of [`htmldocck.py`],
+a supplementary checker script invoked by compiletest that leverages [XPath].
 
+[`htmldocck.py`]: https://github.com/rust-lang/rust/blob/master/src/etc/htmldocck.py
 [XPath]: https://en.wikipedia.org/wiki/XPath
 
-## Directives
-Directives to htmldocck are similar to those given to `compiletest` in that they take the form of `//@` comments.
+## Htmldocck-Specific Directives
 
-In addition to the directives listed here,
-`rustdoc` tests also support most
-[compiletest directives](../tests/directives.html).
+Directives to htmldocck are similar to those given to `compiletest` in that they take the form of `//@` comments.
```

Comment:
Suggestion: even though it's an implementation detail, I think it's worth hammering in the distinction that htmldocck directives are handled by *htmldocck* and *not* compiletest.